a3p-integration/proposals/z:acceptance/test-lib/governance.js

```diff
@@ -55,13 +64,13 @@ export const makeGovernanceDriver = async (fetch, networkConfig) => {
   };
 
   const generateVaultDirectorParamChange = async (
-    previousOfferId,
-    voteDur,
-    params,
-    paramsPath,
+    /** @type {string} */ previousOfferId,
+    /** @type {string | number | bigint | boolean} */ voteDur,
+    /** @type {any} */ params,
+    /** @type {{ paramPath: any; }} */ paramsPath,
```

Comment:
Query:  (I'm not criticizing, I'm trying to understand when this approach should be chosen.)

Why are these declared with in-line types rather than via `@param {type}`?

Reply:
IDE did this for me and I neglected to clean up. will do